src/Sulu/Component/Persistence/EventSubscriber/ORM/UserBlameSubscriber.php

```diff
@@ -18,7 +18,7 @@
 use Sulu\Component\Persistence\Model\UserBlameInterface;
 use Sulu\Component\Security\Authentication\UserInterface;
 use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
-use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
```

Comment:
Another implementation of the tokenStorage currently is used on 4.4-dev of symfony under dev see symfony/symfony@20df3a1.

Reply:
Moved to #4821.